### PR TITLE
Add configurable timeout for accept VPC peering extension

### PR DESCRIPTION
**Pull request by tbroden84**
Failing doing multiple accept VPC peering request, due to firewall still being configured.

- Adds configurable attributes for sleep and timeout.

Dependency: https://github.com/84codes/go-api/pull/20
Issue: https://github.com/cloudamqp/terraform-provider-cloudamqp/issues/152

### CODE REVIEW

File: cloudamqp/resource_cloudamqp_vpc_peering.go [hunk 1]
```diff
@@ -59,7 +72,7 @@ func resourceVpcPeeringAccept(d *schema.ResourceData, meta interface{}) error {
 		d.SetId(d.Get("peering_id").(string))
 	}
 
-	return nil
+	return resourceVpcPeeringRead(d, meta)
```
Comment (dentarg) [inline cloudamqp/resource_cloudamqp_vpc_peering.go:75]: Why this change?

Comment (tbroden84) [reply cloudamqp/resource_cloudamqp_vpc_peering.go:75]: Previous missed to populate status schema attribute during accept VPC peering.
